### PR TITLE
 Add survey profile fixtures

### PR DESCRIPTION
**Pull request by andrew-gardener**

### CODE REVIEW

File: v1p1/caliperEntityMultiselectQuestion.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  "id": "https://example.edu/surveys/100/questionnaires/30/items/4/question",
+  "type": "MultiselectQuestion",
+  "questionPosed": "What do you want to study today?",
+  "points": 4,
+  "itemLabels": ["Calculus", "Number theory", "Combinatorics", "Algebra"],
+  "itemValues": [
+    "https://example.edu/terms/201801/courses/7/sections/1/objectives/1",
+    "https://example.edu/terms/201801/courses/7/sections/1/objectives/2",
+    "https://example.edu/terms/201801/courses/7/sections/1/objectives/3",
+    "https://example.edu/terms/201801/courses/7/sections/1/objectives/4",
```
Comment (ViktorHaag) [inline v1p1/caliperEntityMultiselectQuestion.json]: Trailing comma blows up my JSON de-serializer

File: v1p1/caliperEntityMultiselectResponse.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1/SurveyProfile-extension",
+  "id": "https://example.edu/surveys/100/questionnaires/30/items/5/users/554433/responses/5",
+  "type": "MultiselectResponse",
+  "selections": [
+    "https://example.edu/terms/201801/courses/7/sections/1/objectives/1",
+    "https://example.edu/terms/201801/courses/7/sections/1/objectives/2"
+  ],
+  "startedAtTime": "2018-08-01T05:55:48.000Z",
+  "endedAtTime": "2018-08-01T06:00:00.000Z",
+  "duration": "PT4M12S"
```
Comment (ViktorHaag) [inline v1p1/caliperEntityMultiselectResponse.json]: Missing comma between fields

File: v1p1/caliperEntityOpenEndedResponse.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1/SurveyProfile-extension",
+  "id": "https://example.edu/surveys/100/questionnaires/30/items/2/users/554433/responses/2",
+  "type": "OpenEndedResponse",,
```
Comment (ViktorHaag) [inline v1p1/caliperEntityOpenEndedResponse.json]: Too many commas between fields

**Comment by andrew-gardener** [comment]
fixed issues mentioned

**Comment by ViktorHaag** [comment]
Hey @andrew-gardener and @arwhyte -- now that PR #233 has been merged, and the Survey profile is in the spec develop branch, can we go ahead and move this from draft/WIP into a mergeable PR and merge it?

**Comment by andrew-gardener** [comment]
#238 also adds the `v1p1/caliperEntityCollection.json` fixture. Which should take priority?

Also, I need to do a quick update

**Comment by andrew-gardener** [comment]
I've removed `v1p1/caliperEntityCollection.json` from this PR since I noticed the Collection entity describe was removed from the Survey profile extension spec

**Comment by ViktorHaag** [comment]
Hey @arwhyte  and @andrew-gardener  -- what's stopping us from getting this merged?

**Comment by arwhyte** [comment]
I’m in a van at present heading to Gainesville.  I can review it tonight.

**Comment by ViktorHaag** [comment]
> I’m in a van at present heading to Gainesville. I can review it tonight.

Cool; I got basic broken fixtures in place for all the events we have profiles for. Some of them required porting fixtures over from v1p1, and then I realized that we don't actually yet have any Survey profile fixtures in develop yet. 8)

**Pull request merged**